packages/mini-browser/src/browser/mini-browser-content-style.ts

```diff
@@ -0,0 +1,32 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+export namespace MiniBrowserContentStyle {
```

Comment:
please preserve the original structure and reexport it in mini-browser-content-style.ts to avoid breaking changes, i.e. revert all places to use `MiniBrowserContent.Styles.ERROR_BAR` as before

Reply:
![image](https://user-images.githubusercontent.com/10203793/63676041-b900fd00-c81c-11e9-8979-80a34be963ea.png)
before I use `MiniBrowserContent.Styles.ERROR_BAR` but have error

Reply:
ok, let's go with it, don't think anyone is using it